Sources/Turf/Consumer/Consumer.swift

```diff
@@ -0,0 +1,887 @@
+//
+//  Consumer.swift
+//  Consumer
+//
+//  Created by Nick Lockwood on 01/03/2018.
+//  Copyright © 2018 Nick Lockwood. All rights reserved.
+//
+//  Distributed under the permissive MIT license
+//  Get the latest version from here:
+//
+//  https://github.com/nicklockwood/Consumer
```

Comment:
Can you elaborate on the choice to vendor this library instead of depending on it using a package manager? If we need to avoid an additional third-party dependency, is it possible to accomplish the same functionality using the built-in `Scanner` type from Foundation?

Reply:
My motivation to include the source directly is that this lib is basically a single file, it will not bloat our repo and we most likely will never need to update it's version. Linking through all 3 managers seemed like unnecessary burden.
Built-in features like `Scanner` or `RegEx` did not fit due to minimal OS versions incompatibility. For example majority of required `Scanner` methods require iOS 13.

Reply:
> For example majority of required `Scanner` methods require iOS 13.

`Scanner` is equally capable in iOS 12. All that’s new in iOS 13 are the method overloads for returning an optional type. For example, [`scanFloat(_:)`](https://developer.apple.com/documentation/foundation/scanner/1411677-scanfloat) was introduced in iOS 2.0 as a C-style method that takes an `UnsafeMutablePointer` and returns a Boolean. iOS 13 introduced [`scanFloat(representation:)](https://developer.apple.com/documentation/foundation/scanner/3200284-scanfloat), which returns a `Float?`. The new methods are certainly more convenient, but it should be possible to accomplish the same functionality using the old C-style methods.

Since the current implementation does work fine, we can land this feature based on Consumer. However, we should make its public types private so that downstream packages can continue to depend on Consumer without any conflicts. Consumer is now a fairly large portion of this codebase, so let’s open a separate issue to track replacing Consumer with Scanner as tail work.